agent/stats/container.go

```diff
@@ -78,7 +81,9 @@ func (container *StatsContainer) collect() {
 				// 'NoSuchContainer', 'InactivityTimeoutExceeded' etc are silently consumed.
 				// We rely on state reconciliation with docker task engine at this point of
 				// time to stop collecting metrics.
-				seelog.Debugf("Error querying stats for container %s: %v", dockerID, err)
+				d := backoff.Duration()
```

Comment:
should the backoff be reset when we don't have error?

Reply:
We could, although it's not entirely clear to me why this would ever disconnect without an error, except when the container is terminal, which ends the stats processing completely.

Reply:
i see, then i guess this is probably fine